experiments/enableTTL/README.md

```diff
@@ -0,0 +1,188 @@
+# Effect of `enableTTL`
+
+Dispatcher versions 4.1.11 and greater can be configured to respect a time-to-live (TTL) based content "timeout". TTL headers (`Cache-Control` or `Expires`) must be included in responses sent from the publish tier in order for the dispatcher to know how long a cached file should "live" for in the cache. When this configuration is enabled (and reasonable TTL values are set), there is no longer a need for a dispatcher flush agent.
```

Comment:
Is "there is no longer a need for a dispatcher flush agent" too gentle?  Could you word it to ensure the user knows the flush agent no longer works?  (right?)

Reply:
I'm on the fence about this one. The flush agent still works, but there are implications for validation. I'll add a hint to the section below on "Implications of enableTTL".